onnx/backend/test/cpp/driver_test.cc

```diff
@@ -158,12 +158,14 @@ class ONNXCppDriverTest
                 model_.graph().initializer(0));
         weightDescriptors_pointer = &weightDescriptors;
       }
+      std::string serialized_model;
+      model_.SerializeToString(&serialized_model);
       EXPECT_EQ(
           lib.onnxInitGraph(
               backend,
               NULL,
-              sizeof(model_),
```

Comment:
Did I review this and let it slip through? :)

Reply:
(shhhhhh, lets keep it as a secret